enrolments/serializers.py

```diff
@@ -26,6 +27,14 @@ class Meta:
             "students",
         ]
 
+    def get_phone_number(self, obj):
```

Comment:
since this logic is used in a few places, can we make this a property on the family model and reference that instead? something similar to [this](https://github.com/uwblueprint/project-read-backend/blob/3064e558769f817036cee0d36dc87962a525e85b/registration/models.py#L29-L31) would work!

Reply:
Good point! Updated!